ISM43362Interface.cpp

```diff
@@ -34,6 +34,8 @@ static uint8_t *ism_wifi_thread_stack = NULL;
 
 #define MIN(a,b) (((a)<(b))?(a):(b))
 
+#define ISM43362_WIFI_IF_NAME "is0"
```

Comment:
Why "is0"....?

Reply:
According to multihoming  design every network  interface name should be unique. 
and must contain  two char prefix followed by 8 bit number.  
Prefix can be user selected. I chosed "is" according to first two chars of ISM43362.
In LWIP  first interface 8 bit number is taken from incremented netif counter as new interfaces are added.
ISM43362 uses custom stack and don't  uses netif so this rule cannot be applied.
So  "is0" name  means  ISM43362 - first interface  for ESP8266 it will be "es0".
Currently we don't support custom  stack multiple interfaces  so "0" suffix should be enough.